README.md

```diff
@@ -86,7 +88,9 @@ configuration options. These options can be passed to the agent through the
 object argument to the start command shown below:
 
 ```js
-require('@google-cloud/profiler').start({disableTime: true});
+require('@google-cloud/profiler').start({disableTime: true}).catch((err) => {
```

Comment:
Optional suggestion: instead of making all the examples verbose with a catch block, you could use something like this:

```js
const success = await require('@google-cloud/profiler').start({...});
```

The await succinctly implies that the returned value is a promise. A rejection on the awaited promise will correctly be turned into throw.

Reply:
Maybe even omit the `const success = `.

Reply:
Thanks! Done.